docs/docs/APIdocs/spark.ml/DLClassifier.md

```diff
@@ -0,0 +1,35 @@
+## DLClassifier ##
+
+**Scala:**
+```scala
+val classifier = new DLClassifier[Float](
+    val model: Module[Float],
+    val criterion : Criterion[Float],
+    val featureSize : Array[Int]
```

Comment:
Sure I will comply with the general practice. Yet maybe it will be more meaningful to expose parameter type if we want to elaborate on the constructor. Only exposing the constructor parameter name would not help much.
```
new DLClassifier(
      model: Module[Float],
      criterion: Criterion[Float],
      featureSize: Array[Int])
```
would still compile.